src/components/AdminSettings/MatterbridgeIntegration.vue

```diff
@@ -101,7 +101,7 @@ export default {
 			})
 		},
 		description() {
-			return t('spreed', 'You can install the Matterbridge to link Nextcloud Talk to some other services, visit their {linkstart1}GitHub page{linkend} for more details. Downloading and installing the app can take a while. In case it times out, please install it manually from the {linkstart2}appstore{linkend}.')
+			return t('spreed', 'You can install the Matterbridge to link Nextcloud Talk to some other services, visit their {linkstart1}GitHub page{linkend} for more details. Downloading and installing the app can take a while. In case it times out, please install it manually from the {linkstart2}App Store{linkend}.')
```

Comment:
This is not the Apple App Store and ours is called `appstore` ?!

Reply:
See name: https://github.com/nextcloud/appstore#nextcloud-app-store.

Reply:
@Valdnet Before changing spelling of AppStore/App Store we need to investigate and define the official naming. Just seen your next PRs and want to stop you 😄 .

Reply:
@rakekniven Well. How to determine the correct name (with whom)? You know it will be about improving all text strings with the name of the store 😁.

Reply:
> @rakekniven Well. How to determine the correct name (with whom)? You know it will be about improving all text strings with the name of the store 😁.

I will write a topic at the forum and ask anyone for input and then Marketing department should make a decision.
I know it will be work afterwards to implement the decision.